db/migrate/20231025055905_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.references :author, foreign_key: { to_table: :users }, index: true
+      t.string :title
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+      t.timestamps
```

Comment:
- [ ] As required for the project and according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given to you, kindly ensure that the `likes` have the same columns like in the ERD diagram. I noticed that you made confusion between table names. Make sure you fix that